### PR TITLE
Add missing include of inttypes.h in rf-ble-cmd.c

### PR DESCRIPTION
**Pull request by arurke**
As in title.

### CODE REVIEW

**Comment by arurke** [comment]
@pjonsson Are you familiar with the failures in all the mac os jobs? A quick google did not offer me much.

**Comment by pjonsson** [comment]
@arurke it seems the Github runners got a new macOS version, and that switched them from Intel to M1 silicon. I'm just heading into a meeting, can you please make a PR here that switches from macos-latest to the older version used by the Cooja repository? It's not just the matrix that needs to be updated, there are also checks further down that needs to be updated, brew for example is only executed on macOS.

**Comment by nfi** [comment]
I made a PR to address this issue: #2890

File: arch/cpu/cc26x0-cc13x0/rf-core/ble-hal/rf-ble-cmd.c [hunk 1]
```diff
@@ -45,6 +45,7 @@
 #include "rf_ble_cmd.h"
 #include "rf-core/rf-core.h"
 #include "rf-core/ble-hal/rf-ble-cmd.h"
+#include <inttypes.h>
```
Comment (pjonsson) [inline arch/cpu/cc26x0-cc13x0/rf-core/ble-hal/rf-ble-cmd.c:48]: If this file is missing, shouldn't the include be in `rf_ble_cmd.h` instead? There are plenty of uses of `uint8_t` in the function prototypes.

Comment (arurke) [reply arch/cpu/cc26x0-cc13x0/rf-core/ble-hal/rf-ble-cmd.c:48]: iirc, it was the `PRIX32` on line 525 that I got a complaint about. I am guessing `stdint.h` is carried with some of the other includes in `rf_ble_cmd.h` and that is why it passes. But in any case, you are right -  I added `stdint.h` to the header-file (as opposed to `inttypes.h`) to keep it as targeted as possible.

Comment (pjonsson) [reply arch/cpu/cc26x0-cc13x0/rf-core/ble-hal/rf-ble-cmd.c:48]: Oh, for `PRIX32` you are right about the header, nevermind my comment then.

**Review by pjonsson — APPROVED** [review]

**Review by nfi — APPROVED** [review]

**Pull request merged**